lib/widgets.js

```diff
@@ -37,6 +37,9 @@ var input = function (type) {
                 classes: w.classes,
                 value: w.formatValue(f.value)
             };
+            if (f.placeholder !== undefined) {
```

Comment:
Why is this needed? `placeholder` should already be supported from line 23 above (`getUserAttrs(opts)`).